WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewController.swift

```diff
@@ -0,0 +1,180 @@
+import UIKit
+import Yosemite
+import WordPressUI
+import Combine
+
+/// Displays a list of settings for the user to choose to bulk update them for all variations
+///
+final class BulkUpdateViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private let viewModel: BulkUpdateViewModel
+
+    private var subscriptions = Set<AnyCancellable>()
+
+    init(viewModel: BulkUpdateViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTableView()
+        configureViewModel()
+    }
+
+    /// Setup receiving updates for data changes and actives the view model
+    ///
+    private func configureViewModel() {
+        viewModel.$syncState.sink { [weak self] state in
+            guard let self = self else { return }
+
+            switch state {
+            case .notStarted:
+                fallthrough
```

Comment:
Why did you decide to do a `fallthrough` here? I think `.notStarted` should do anything.

Reply:
`syncing` is triggered when the view is loaded, my thinking was this could change in the future and have an other condition trigger the `syncing`(e.g. waiting a future with the variations loading from the previous screen), so we should given a visual feedback to the user until then.
But I have reconsider this, if in the future we change the VM logic we should also change the VC if needed. I have updated the PR so `.notStarted` does nothing.